src/bacnet/basic/object/nc.c

```diff
@@ -444,7 +444,7 @@ bool Notification_Class_Write_Property(BACNET_WRITE_PROPERTY_DATA *wp_data)
                     (tt_hour > 23 || tt_min > 59 || tt_sec > 59)) {
                     wp_data->error_class = ERROR_CLASS_PROPERTY;
                     wp_data->error_code = ERROR_CODE_VALUE_OUT_OF_RANGE;
-                    break;
+                    return false;
```

Comment:
consider status = false; break; - just make sure that status = true; @ line 461 doesn't get executed....

Reply:
agreed with the above suggestion

Reply:
yup so if you would move status = true from line 461 to line from before the while loop it would work; break will exit the loop; the the break after the loop will exit the switch...